.github/workflows/tests.yml

```
@@ -99,12 +96,7 @@ jobs:
     - name: Run brew readall on all taps
       run: brew readall --aliases
 
-    - name: Run brew style on homebrew-core
-      if: matrix.os == 'macOS-latest'
-      run: brew style --display-cop-names homebrew/core
-
-    - name: Run brew style on linuxbrew-core
-      if: matrix.os == 'ubuntu-latest'
+    - name: Run brew style on ${{ matrix.core-tap }}
```

```suggestion
    - name: Run brew style on ${{ matrix.config.core-tap }}
```

And similarly for `matrix.os`.

---

Thanks, @Bo98 -- forgot about that... Now that this is fixed, I'm not sure whether this is an improvement or not :)